src/components/change-requests/change-request-details/change-request-details/review-notes/review-notes.tsx

```diff
@@ -3,20 +3,37 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 import { User } from 'utils';
-import { fullNamePipe } from '../../../../../shared/pipes';
+import { datePipe, fullNamePipe } from '../../../../../shared/pipes';
 import PageBlock from '../../../../shared/page-block/page-block';
 
 interface ReviewNotesProps {
   reviewer?: User;
   reviewNotes?: string;
+  dateReviewed: Date;
 }
 
-const ReviewNotes: React.FC<ReviewNotesProps> = ({ reviewer, reviewNotes }: ReviewNotesProps) => {
+const ReviewNotes: React.FC<ReviewNotesProps> = ({
+  reviewer,
+  reviewNotes,
+  dateReviewed
+}: ReviewNotesProps) => {
   return (
     <PageBlock
       title={'Review Notes'}
-      headerRight={<>{fullNamePipe(reviewer)}</>}
+      headerRight={
+        <OverlayTrigger
```

Comment:
the `fullNamePipe` actually handles the input potentially being undefined by returning an em-dash instead of the name. I can't remember if the `datePipe` handles undefined or if the `emDashPipe` still exists, but basically we still want to return an em-dash if the change request has not been reviewed. "Reviewed on: <em-dash>" is perfectly acceptable for a tooltip content when the change request has not been reviewed.